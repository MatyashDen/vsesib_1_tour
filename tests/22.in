8 10 3 10
0 4
0 8
2 9
3 4
4 8
5 5
7 3
7 4
7 6
7 8
