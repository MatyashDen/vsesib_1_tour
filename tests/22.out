620
0RLE0LRE0DUE0RRLLE0RDULE0LLRRE0LDURE0DDUUE0RRDULLE0RDDUULE0LLLRRRE0LLDURRE0LDDUURE0DDDUUUE0RDDDUUULE0LLLLRRRRE0LLLDURRRE0LLDDUURRE0LDDDUUURE0DDDDUUUUE0RDDDDUUUULE0LLLLDURRRRE0LLLDDUURRRE0LLDDDUUURRE0LDDDDUUUURE0DDDDDUUUUUE0RDDDDDUUUUULE0LLLLDDUURRRRE0LLLDDDUUURRRE0LLDDDDUUUURRE0LDDDDDUUUUURE0DDDDDDUUUUUUE0RDDDDDDUUUUUULE0LLLLDDDUUURRRRE0LLLDDDDUUUURRRE0LLDDDDDUUUUURRE0LDDDDDDUUUUUURE0DDDDDDDUUUUUUUE0RDDDDDDDUUUUUUULE0LLLLDDDDUUUURRRRE0LLLDDDDDUUUUURRRE0LLDDDDDDUUUUUURRE0LDDDDDDDUUUUUUURE0LLLLDDDDDUUUUURRRRE0LLLDDDDDDUUUUUURRRE0LLDDDDDDDUUUUUUURRE0LLLLDDDDDDUUUUUURRRRE0LLLDDDDDDDUUUUUUURRRE0LLLLDDDDDDDUUUUUUURRRRE
1RLE1LRE1DUE1LDUREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
2LRE2DUE2UDE2LLRRE2LDURE2DDUUE2LLLRRRE2LLDURRE2LDDUURE2DDDUUUE2LLLDURRRE2LLDDUURRE2LDDDUUURE2DDDDUUUUE2LLLDDUURRRE2LLDDDUUURRE2LDDDDUUUURE2DDDDDUUUUUE2LLLDDDUUURRRE2LLDDDDUUUURRE2LDDDDDUUUUURE2LLLDDDDUUUURRRE2LLDDDDDUUUUURRE2LLLDDDDDUUUUURRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
